### ISSUE TITLE
High-performance context causes some versions of Safari to fail to load

### ISSUE THREAD

**Issue by OmarShehata** [CesiumGS/cesium#8693]
This was reported [on the forum](https://groups.google.com/forum/?hl=en#!topic/cesium-dev/Adihtx_M4Ow) for Safari 13.05 on High Sierra 10.13.6. Opening any CesiumJS app, the page will continually start to refresh and then finally say "A problem repeatedly occurred".

This only happens starting 1.66, when the powerPreference was set to `high-performance`. This is fixed by setting the context options to `default` or `low-power` as suggested here:

https://github.com/CesiumGS/cesium/blob/master/CHANGES.md#additions-tada-2

Others have tested with Safari on OSX and have not encountered this.

**Comment by mramato** [comment]
This sounds like a Safari or MacOS bug on an outdated version or a machine-specific issue.  Does the problem happen in Chrome or Firefox on the same machine? Does other webgl content work?  What GPUs does the machine have?

> the page will continually start to refresh and then finally say "A problem repeatedly occurred".

This is standard Safari behavior whenever a page crashes the Safari tab itself, if you have the debugger open with exceptions on, you can usually get more information before it refreshes. Not sure if you can turn the refreshing off.

> This only happens in 1.66

Did you mean the problem doesn't start until 1.66, or does it not happen in 1.67?

**Comment by OmarShehata** [comment]
> This sounds like a Safari or MacOS bug on an outdated version or a machine-specific issue.

Yeah. Should we close this issue? There's nothing actionable for us other than to consider not having high-performance as the default, but that doesn't sound like a good idea just to get around a bug in a specific version of a specific browser.

> Does the problem happen in Chrome or Firefox on the same machine?

Only happens in Safari for that user. Chrome and Firefox were verified to work.

>  Does other webgl content work?

ThreeJS was tested on that machine, the examples/demos there work for the user.

> What GPUs does the machine have?

intel hd graphics 630. Full WebGL report from user:

[WebGl_Report_as@iidj.net.pdf](https://github.com/CesiumGS/cesium/files/4354816/WebGl_Report_as%40iidj.net.pdf)

> Did you mean the problem doesn't start until 1.66, or does it not happen in 1.67?

I meant starts on 1.66 and after. I'll reword it to make that clear.

**Comment by mramato** [comment]
Unfortunately, I agree that there's nothing actionable here unless we can reproduce ourselves.  I vote we close it and re-open it the moment anyone else starts having the issue or we find a way to reproduce it.

**Comment by cesium-concierge** [comment]
Congratulations on closing the issue! I found these Cesium forum links in the comments above:

https://groups.google.com/forum/?hl&#x3D;en#!topic/cesium-dev/Adihtx_M4Ow

If this issue affects any of these threads, please post a comment like the following:

> The issue at https://github.com/CesiumGS/cesium/issues/8693 has just been closed and may resolve your issue. Look for the change in the next stable release of Cesium or get it now in the master branch on GitHub https://github.com/AnalyticalGraphicsInc/cesium.
